episodes/reviewing_pull_requests.md

```diff
@@ -13,47 +13,62 @@ exercises: 0
-- to comment on a range of lines, click-and-hold on the `+` for the first line in the range, and drag your cursor down the range, then release
+## Title and description
+A good title and description for a pull request should give the reviewer an immediate impression of the goal(s) and scope of the changes it contains.
+[Pull request templates, covered later](./collaborating_newcomers.md#guide-contributors-with-issue-and-pull-request-templates), are one way to help contributors provide this information.
 
-## Suggesting Changes
+## Line-by-line comments
+The specific changes proposed in a pull request can be viewed in its _Files Changed_ tab.
+It is usually most helpful to provide feedback and ask questions on the lines that have been changed.
+You can comment on a particular line by hovering the cursor over it and clicking the `+` button on the left-hand side.
+Comments can be added on their own, or as part of a _Review_: a commentary on the pull request as a whole. When choosing the latter, comments are only published after the reviewer has finished the whole review.
+To comment on a range of lines, click-and-hold the `+` button for the first line in the range, drag your cursor down the range, then release.
 
-- as well as adding comments line-by-line, you can also suggest specific changes, e.g. to fix a typo, suggest improved wording, etc.
-- take the same approach to start a comment on a line or range of lines, then click the "add a suggestion" button, and edit the source content that appears in the input box
-- this makes it much easier for contributors to a) understand reviewers' suggestions, and b) implement them
+## Suggesting changes
+As well as adding comments line-by-line, you can also suggest specific changes, e.g. to fix a typo, suggest improved wording, etc.
+To do this, take the same approach to start a comment on a line or range of lines, then click the "add a suggestion" button (plus/minus symbol inside a document icon), and edit the source content that appears in the input box with your suggested fix/improvement.
+Suggesting specific changes makes it much easier for contributors to a) understand your suggestions, and b) implement them.
+Suggested changes will be commited to the pull request branch as co-authored by the pull request author and reviewer.
 
-## Finishing a Review
+## Finishing a review
+When you have finished commenting on particular lines, and suggesting changes, you can finish your review by clicking the "Review changes" or "Finish your review" button near the top-right of the screen.
+This is the place to add general comments and questions, and to summarise your review.
+If nothing else, do not forget to thank the contributor for taking the time to contribute!
+Choose one of the three options:
+
+- "Comment" - a general comment, with no firm outcome. Can be used to complete a review before a pull request is closed.
```

Comment:
I'm confused about the "before a pull request is closed" part.  Is that just saying you can't comment before a PR is closed?  Isn't that true for approving and rejecting too?